Frontend/src/pages/admin/AdminCreateNotificationPage.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <q-card flat bordered id="card">
+    <q-card-section>
+      <div class="text-h6">Send Notification</div>
+    </q-card-section>
+    <div class="q-pa-md">
+      <q-table
+        title="Select Students"
+        :data="tableStudents"
+        :columns="columns"
+        row-key="email"
+        selection="multiple"
+        :selected.sync="toSendTo"
+      />
+      <q-input filled v-model="title" label="Title" style="margin-top: 3%" />
+      <q-input
+        v-model="body"
+        label="Body"
+        filled
```

Comment:
Make the inputs `outlined` instead of `filled` to match the style of the rest of the site